modules/core/src/main/scala/higherkindness/droste/basis.scala

```diff
@@ -18,6 +18,10 @@ import cats.Monoid
 import cats.free.Trampoline
 import cats.instances.function._
 
+/** An `Embed[F, R]` is just a trait that provides an `Algebra[F, R]`,
+  * which is an alias for a `GAlgebra[F, A, A]`, which is itself 
```

Comment:
This explanation seems a bit pedantic.

How about something like:
"`Embed[F, R]` provides implicit evidence of an `Algebra[F, R]` for use in unfolds to structure `R`."